workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/src/components/ui/ui.style.ts

```diff
@@ -0,0 +1,200 @@
+import { makeStyles } from '@material-ui/core';
+import { Theme } from '@mui/material/styles';
+
+export const useCustomHeaderStyles = makeStyles(() => ({
+  '@font-face': {
+    fontFamily: 'InterFont',
+    src: `url('../../assets/fonts/Inter/Inter-Regular.ttf') format('truetype')`,
+  },
+  organizationPickerContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    height: '20px',
+    width: '170px',
+    marginRight: '30px',
+  },
+  root: {
+    fontFamily: 'InterFont',
+  },
+  container: {
+    width: '100%',
+    position: 'relative',
+    fontFamily: 'InterFont, sans-serif !important',
+    '& >div': {
+      boxShadow: 'none',
+    },
+  },
+  registerBtn: {
+    color: 'white',
+    backgroundColor: '#219DB6',
```

Comment:
Colors should not be hard coded and should be coming from the theme. 👍

Reply:
That goes with all cases in this file and anywhere else

Reply:
Done👍